docs/index.rst

```diff
@@ -13,7 +13,7 @@ of proprietary formats to the Neurodata Without Borders (NWB) standard.
 
 Features:
 
-* Reads data from 40 popular neurophysiology data formats and writes to NWB using best practices.
+* Reads data from 42 popular neurophysiology data formats and writes to NWB using best practices.
```

Comment:
Where did these other 2 come from? We recently bumped from 38 due to Bruker and MicroManager